docs/source/6_api_reference/api.rst

```diff
@@ -56,3 +56,10 @@ misc
 
 .. automodule:: shimmingtoolbox.cli.referencemaps
    :members:
+
+cli
+---
+
+.. click:: shimmingtoolbox.cli.mask:mask_cli
+   :prog: mask
+   :nested: full
```

Comment:
We have a new section for CLI documentation [here](https://github.com/shimming-toolbox/shimming-toolbox/blob/master/docs/source/6_cli_reference/cli.rst) - could you move these stubs into there?

Reply:
Done now, thank you for the information !